_docs/other/community-geyser-projects.md

```diff
@@ -25,19 +25,27 @@ A Spigot shop plugin that uses Bedrock forms.
 
 Creator: [KejonaMC](https://github.com/kejonaMC)
 
+## Bedrock Player Manager for Floodgate
+A Spigot plugin that is able to run certain commands when a Bedrock player joins, or have Bedrock players able to bypass the anticheat, etc. 
+
+* [Spigot page](https://www.spigotmc.org/resources/82278/)
+
+Creator: ofunny
+
 ## BedrockSkinUtility
 This is a Fabric mod that allows you to view Bedrock capes from Java.
 
 * [GitHub](https://github.com/Camotoy/BedrockSkinUtility)
 
 Creator: [Camotoy](https://github.com/Camotoy)
 
-## Bedrock Player Manager for Floodgate
-A Spigot plugin that is able to run certain commands when a Bedrock player joins, or have Bedrock players able to bypass the anticheat, etc. 
+## CreeperNation Skin API
 
-* [Spigot page](https://www.spigotmc.org/resources/82278/)
+an SkinAPI that displays Java or Bedrock skins based on the UUID or username
```

Comment:
Captial. and a dot at end of sentence. :)

Reply:
Done